LearningJava/src/a_basics_Of_Java/a1_DataTypesAndSize.java

```diff
@@ -13,6 +13,6 @@ public class a1_DataTypesAndSize {
 	 * char  -  2 bytes
 	.............. */
 	
-	
+	//Adding comment to test the synch and fetch
```

Comment:
Jai, what is the purpose of adding this comment

Reply:
This is just to test thats it